ml_synth_multi_saw_example.ino

```diff
@@ -213,6 +213,12 @@ void setup()
     /* PSRAM will be fully used by the looper */
     Serial.printf("Total PSRAM: %d\n", ESP.getPsramSize());
     Serial.printf("Free PSRAM: %d\n", ESP.getFreePsram());
+
```

Comment:
Good point, I think it would be better to prepare a general function like Board_Setup or similar. This will be put in the library to avoid changing all existing projects

Reply:
I've checked my files. I would prefer to Move the WM8978_setup routine into the audio setup and maybe put a #ifdef WM8778_ENABLED around it.

Reply:
I've prepared an update to make the WM8978 module available for global usage. Please pull the update. Do you think it would be a good solution to put the definition into wm8978.h ?

Reply:
Yes. I prefer a clean setup as in a well defined board.h for each board. The pins are fixed on those boards and a lot of them share the same hardware and functions but with different pin definitions. 

A "common_board.h" could be the place holder for people to define pins for there own build and by doing it that way your synth tool box would be a lot easier to adapt on different hardware DIY or bought.

At the moment pin definitions is a bit "all over the place" no hard feelings :)
I had a lot of issues on just figuring out what pin definition was the "true" one as the Arduino IDE sucks on those basic parts. Cloning and using PlatformIO made it possible to trace back where the changes needed to be done

Thanks for a greate lib. Just ordered my first eurorack case and MIDI interfaces.
Your lib will soon have a sweet home at my desk

Reply:
You are right, I did some changed over the time to find a clean concept to find out where to place the information. My goal is to separate project and general information. A project shouldn't care about different boards etc. It can be configured to enable/disable features.

Having a DIY setup pin definition can be part of the project. If there is an official board / PCB used with a fixed PIN setup, the definition goes into the board header file.

I am happy that it finds a new home =)

Reply:
I have a  idea. When I confirmed my setup on the TTGO T9 board I just made "my_custom_bord.ini" where I put everything, used your defines and added mine. 
I could make a framework for ESP32 following your layout on bords.

With well documented comments I think it would help people. And if there is a new board in town, save it with the board name and then just merge it in to the lib
//Agge